rdflib/compat.py

```diff
@@ -2,9 +2,9 @@
 Utility functions and objects to ease Python 2/3 compatibility,
 and different versions of support libraries.
 """
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+
+
+
```

Comment:
Could you please also remove the empty lines here and in other similar cases?

Reply:
Sure. I am not happy with that neither. Will commit that soon...